ios/edge/AppDelegate.m

```diff
@@ -128,9 +128,9 @@ - (void)application:(UIApplication *)application
       }
 
       UNMutableNotificationContent *content = [[UNMutableNotificationContent alloc] init];
-      content.title = @"New Device Login";
+      content.title = @"Urgent Security Issue";
       content.body = message;
-      UNTimeIntervalNotificationTrigger* trigger = [UNTimeIntervalNotificationTrigger
+      UNTimeIntervalNotificationTrigger* trigger = [UNTimeIntehervalNotificationTrigger
```

Comment:
UNTimeIntehervalNotificationTrigger (sp). This should have caused a compile error

Reply:
I blame the toddler for extra keystrokes.